src/game/protos/object.c

```diff
@@ -146,6 +148,19 @@ int object_serialize(object *obj, serial *ser) {
     return 0;
 }
 
+int object_clone(object *src, object *dst) {
+    memcpy(dst, src, sizeof(object));
+    src->clone(src, dst);
+    if (src->custom_str) {
+        dst->custom_str = strdup(src->custom_str);
+    }
+
+    if (src->clone) {
```

Comment:
so here is a NULL-check for src->clone, but there the function is called before that, possibly crashing?

Reply:
this is for the object's specializer clone call, not the generic object clone